modules/installation-requirements-user-infra.adoc

```diff
@@ -93,22 +97,25 @@ Each cluster machine must meet the following minimum requirements:
 |120 GB
 
 |Compute
-|{op-system}
+ifdef::ibm-z[|{op-system}]
+ifndef::ibm-z[|{op-system} or RHEL 7.6]
 |2
 |8 GB
 |120 GB
 
 |===
 
-ifeval::["{context}" == "installing-ibm-z"]
+ifdef::ibm-z[]
 [id="minimum-ibm-z-system-requirements_{context}"]
 == Minimum IBM Z system requirements
 
+You can install OpenShift 4.2 on IBM Z hardware as of z13, z13s, or LinuxONE.
```

Comment:
s/OpenShift 4.2/{product-title} version {product-version}

"IBM Z hardware as of z13, z13s, or LinuxONE" doesn't scan right to me. Does "on the following IBM Z hardware types: z13, z13s, or LinuxONE" sound right to you?

Reply:
I'd change it to: "You can install OpenShift 4.2 on IBM LinuxONE or one of the following IBM Z hardware products: z13, z13s." 

Wdyt @kalexand-rh @dbenoit17?

Reply:
I think that looks good!

Reply:
@dbenoit17, does this look ok to you?

> You can install OpenShift 4.2 on IBM LinuxONE or one of the following IBM Z hardware products: z13, z13s.

Reply:
Rephrased in 36c8aa640b1c0301f102b11713b57ea33b720242 
Maria wanted to add the HW as it was missing and this was her suggestion. I would add more HW later if needed, but for now we have the info there, which is imporant.

Reply:
We also support the z14 and z15 platforms.  Could we add those to the list?

Thanks!